test/parallel/test-require-deps-deprecation.js

```diff
@@ -23,9 +23,12 @@ const deprecatedModules = [
 
 // Newly added deps that do not have a deprecation wrapper around it would
 // throw an error, but no warning would be emitted.
+// Added false dep to check test throws even if acorn is in the user's
+// resolve tree.
```

Comment:
This wouldn't be necessary.

Reply:
Do you mean the comment, or the 'foo' dep too?

Reply:
The “foo” dep… and as a result the comment too.